websurfx/config.lua

```diff
@@ -73,3 +73,5 @@ upstream_search_engines = {
     Mojeek = false,
     Bing = false,
 } -- select the upstream search engines from which the results should be fetched.
+
+proxy = "" -- Proxy to send outgoing requests through. Set to empty string to disable.
```

Comment:
```suggestion
proxy = "socks5h://127.0.0.1:9050" -- proxy string to send outgoing requests through. 
```

I would suggest changing this to use the localhost address with port 9050 by default and changing the explaination to exclude the empty string statement because the user can disable/enable this option using the `socks` feature during the build stage. 🙂